src/compiletest/runtest.rs

```diff
@@ -1013,8 +1013,8 @@ fn check_expected_errors(revision: Option<&str>,
         expected_errors.iter()
                         .fold((false, false),
```

Comment:
Pre-existing, but can't we use `any` here instead of `fold`? Seems like it more clearly expresses the intent.